### ISSUE TITLE
Average Column Charts display Incorrect Values since upgrading to 1.3b1 ( is it #589,#590? )

### ISSUE THREAD

**Issue by michaelundwd** [poblabs/weewx-belchertown#616]
**First step**
I have used the graphs.conf.example and it works perfectly, so I know the issue lies with the content of my graphs.conf file. 

**Describe the bug**
Since upgrading to 1.3b1, my column charts display the average values incorrectly.  They are calculating the sum, rather than the average values of totalRain, radiation, maxSolarRad and UV.  I attach two images showing the before and after the update, showing the differences in the radiation chart.  The totalRain is incorrect in both versions.

**Link to your website (IMPORTANT)**
http://lordshipweather.uk

**Version of the skin you're using**
1.3b1

**To Reproduce**
Visit http://192.168.2.100/live/graphs/?graph=OverallSummary

**Expected behavior**
See the two screenshots

**Screenshots**

**Device Information (please complete the following information):**
browser-independent

**Additional context**
Relevant section of graphs.conf attached as Excerpt_graphs.zip

[Excerpt_graphs.zip](https://github.com/poblabs/weewx-belchertown/files/6063687/Excerpt_graphs.zip)
![Screenshot 2021-03-01 170618_test](https://user-images.githubusercontent.com/45148992/109542077-817faa80-7abc-11eb-8259-ce170dd80e15.png)
![Screenshot 2021-03-01 171806_live](https://user-images.githubusercontent.com/45148992/109542080-82184100-7abc-11eb-8e57-3afd10dff825.png)

**HELP WANTED**
If you have a fix for this, please submit a pull request against the development branch!

**Comment by norecords** [comment]
I've got the same issue, example: 
```
        [[[outTemp_avg]]]
            name = Température moy.
            observation_type = outTemp
            aggregate_type = avg

```
look like render sum of values.

**Comment by michaelundwd** [comment]
@norecords
Is your database binding for mysql or sqlite?  I'm using sqlite.

**Comment by norecords** [comment]
I'm using mysql

**Comment by michaelundwd** [comment]
I'm going to try taking out the mods for using the daily summary introduced in belchertyown.py in #589 and see if that cures the problem.

**Comment by michaelundwd** [comment]
That has fixed my annual solar column charts, so points to something not quite right with the code to use the daily summaries.
You might like to try the mysql code.  I went back to the version in https://github.com/poblabs/weewx-belchertown/commit/b254916ca930568c72387f60b79f881acd008aa7 and just copied the code starting ~line 3343
if driver == "weedb.sqlite":

**Comment by michaelundwd** [comment]
solved  in PR #619
